elastic/kibana
# [Security Solutions] Fix User flyout loading forever when Managed user flag is enabled

## Summary

* Fix the bug that made the user modal load forever
* Add unit tests
* Add cypress test for managed data
* Flaky test runner https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/4975

### How to test it?
* Enable `newUserDetailsFlyoutManagedUser` and `newUserDetailsFlyout` flags
* Open a kibana instance with alerts
* Go to Alerts page and click on a username
* It should load the new user details flyout


### Checklist
